app/javascript/entry_points/publicListings.js

```diff
@@ -1,4 +1,4 @@
-import Vue from 'vue'
+import Vue from 'our_vue'
```

Comment:
maybe `buefy_vue` ?

Reply:
Yeah i don't love the name `our_vue`, but we'll likely further configure our Vue instance with other plugins and such.